library/analytics/src/main/java/org/cru/godtools/analytics/model/AnalyticsActionEvent.kt

```diff
@@ -14,20 +13,13 @@ open class AnalyticsActionEvent(
     locale: Locale? = null,
     systems: Collection<AnalyticsSystem>? = null
 ) : AnalyticsBaseEvent(locale, systems) {
-    companion object {
-        fun String.sanitizeAdobeNameForFirebase(): String = replace(Regex("[ \\-.]"), "_").toLowerCase(Locale.ROOT)
-    }
-
     constructor(action: String, label: String? = null, locale: Locale? = null, system: AnalyticsSystem) :
         this(action, label, locale, setOf(system))
 
     open val adobeAttributes: Map<String, *>? get() = null
 
     open val firebaseEventName get() = action
-    open val firebaseParams
-        get() = Bundle().apply {
-            adobeAttributes?.forEach { putString(it.key.sanitizeAdobeNameForFirebase(), it.value?.toString()) }
-        }
+    open val firebaseParams: Map<String, *>? get() = null
```

Comment:
I actually want to keep `firebaseParams` returning a `Bundle` since the firebase track event method takes a bundle directly. It provides more direct access to how additional attributes are passed to firebase without needing to assume all values should be stringified.

Reply:
Yeah I was trying to find a way to pass the values in order to map through them in the bundle but couldn't figure out an approach that wasn't either making it a function or using a separate value like we are currently doing(ex. ```adobeAttributes```) .

Reply:
We can work through it next week when I'm back

Reply:
Sounds good 🙂

Reply:
instead of changing `firebaseParams` the focus should be on removing usages of `adobeAttributes` by converting them to return a `Bundle` for `firebaseParams` instead.

Reply:
Ah so make ```firebaseParams``` be equal to ```adobeAttributes``` but change ```adobeAttributes``` to return a bundle instead?

Reply:
not exactly, I'll send you a google meet request

Reply:
`adobeAttributes` should be what is being refactored out